pkg/engine/task/task_apply.go

```diff
@@ -93,6 +99,16 @@ func apply(ro []runtime.Object, c client.Client) ([]runtime.Object, error) {
 	return applied, nil
 }
 
+func isClusterResource(r runtime.Object) bool {
+	switch r.(type) {
+	case *apiextv1beta1.CustomResourceDefinition:
+		return true
+	case *v1.Namespace:
```

Comment:
@akirillov already mentioned `ClusterRoles` so I guess we should add cases for everything we can think of (see @alenkacz comment above).

Reply:
yeah... as I slept last night I thought we should put all the things in here... +1